### PR TITLE
Feature/category products

### PR DESCRIPTION
**Pull request by michalskoczek**
I have a problem with star rating, maybe some advice?? :)

### CODE REVIEW

File: src/app/components/category-products/category-products.component.html [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+
+        </ng-container>
+      </div>
+    </div>
+  </div>
+</section>
+
+
+<footer class="footer mt-5">
```
Comment (gradzio) [inline src/app/components/category-products/category-products.component.html]: i think it is a duplicated code fragment - what is the better place to put it?

File: src/app/components/category-products/category-products.component.html [hunk 2]
```diff
@@ -0,0 +1,222 @@
+<header>
```
Comment (gradzio) [inline src/app/components/category-products/category-products.component.html]: this is a code duplication, where can you put it then?

**Review by gradzio — CHANGES_REQUESTED** [review]

File: src/app/components/category-products/category-products.component.html (hunk 1 above)
Comment (michalskoczek) [reply src/app/components/category-products/category-products.component.html]: I use other component

File: src/app/components/category-products/category-products.component.html (hunk 2 above)
Comment (michalskoczek) [reply src/app/components/category-products/category-products.component.html]: I use other component

**Pull request merged**